plugins/vlc-video/vlc-video-source.c

```diff
@@ -436,6 +438,11 @@ static void add_file(struct vlc_source *c, struct darray *array,
 					network_caching_option.array);
 			dstr_free(&network_caching_option);
 		}
+		struct dstr track_option = { 0 };
+		dstr_catf(&track_option,
+			":audio-track=%d", track_index - 1);
```

Comment:
Style:  Tab twice to indent a line continuation (see line 434->436 above).

```suggestion
				":audio-track=%d", track_index - 1);
```

Reply:
done